### DISCUSSION TITLE
what about prefix timestamp to Log records and read min, max, range voltage ?

### DISCUSSION THREAD

**Discussion by ornolio** [fl4p/batmon-ha#62] [Ideas]
sometime it could be useful to know timestamp of  errors or calculate waiting time in discovery to better tune timing parameters setup.

Actually I calculate the BMS voltage: min, max and range adding some hassio Assistants. Doesn't those values directly readable from jbd BMS?

**Comment by fl4p** [comment]
Good idea, I'll add timestamps with the next update.
Also planning to add min, max and delta Voltages

**Comment by fl4p** [comment]
Added min, max and delta for the next release
